explicit-requests/README.md

```diff
@@ -0,0 +1,59 @@
+# Explicit Requests
```

Comment:
"Custom Requests"?

Reply:
this example shows the "complicated" use-case of multiple, parallel requests. let's split it into two chapters: 
- use a custom `request` (no proxy)
- multiple parallel requests